code/modules/mod/modules/modules_supply.dm

```diff
@@ -428,6 +428,8 @@
 	mod.wearer.add_filter("mod_outline", 3, outline_filter(color = "#000000AA"))
 	animate(mod.wearer, animate_time, pixel_y = mod.wearer.pixel_y - 4, flags = ANIMATION_PARALLEL)
 	mod.wearer.SpinAnimation(1.5)
+	// todo, someone get balance approval to add TRAIT_FORCED_STANDING here, like it is on tg.
```

Comment:
Someone 🤣

Reply:
out of scope :), but it would fix things

Reply:
Just get a PR up to handle it after this one goes in